### PR TITLE
Fix #88,#30 - hardcode Bugzilla repos and warn if no Bugzilla issue is referenced

### LINKED ISSUE #88: Only check Bugzilla for dub - only for dmd, druntime, phobos, tools, installer, dlang.org
e.g. https://github.com/dlang/dub/pull/1164

### PR DESCRIPTION
**Pull request by wilzbach**
Two flies with one clap :)

### CODE REVIEW

File: source/dlangbot/github.d [hunk 1]
```diff
@@ -57,11 +57,26 @@ Please see [CONTRIBUTING.md](https://github.com/%s/blob/master/CONTRIBUTING.md)
...
     {
-        app ~= "### Bugzilla references\n\n";
-        app.printBugList(refs, descs);
+        app ~= "\n### Bugzilla references\n\n";
+        if (refs.length)
+            app.printBugList(refs, descs);
+        else
+            app.formattedWrite(
+`Your PR doesn't reference any Bugzilla issue.
+<p>
```
Comment (wilzbach) [inline source/dlangbot/github.d]: @ZombineDev: seems like this is the only way to break long lines in Markdown ... ugly :cry:

File: source/dlangbot/github.d [hunk 2]
```diff
@@ -57,11 +57,26 @@ Please see [CONTRIBUTING.md](https://github.com/%s/blob/master/CONTRIBUTING.md)
...
+If your PR contains non-trivial changes, please
+<a href="https://github.com/dlang-bots/dlang-bot#automated-references">reference a Bugzilla issue</a>
+or create a <a href="https://github.com/%s/blob/master/changelog">manual changelog</a>.
```
Comment (ZombineDev) [inline source/dlangbot/github.d]: Can't we use the markdown syntax here?

File: source/dlangbot/github.d [hunk 3]
```diff
@@ -57,11 +57,26 @@ Please see [CONTRIBUTING.md](https://github.com/%s/blob/master/CONTRIBUTING.md)
 
 `, pr.user.login, pr.repoSlug);
 
-    if (refs.length)
+    static immutable bugzillaReferences = ["dlang/dmd", "dlang/druntime", "dlang/phobos",
+                     "dlang/dlang.org", "dlang/tools", "dlang/installer"];
```
Comment (ZombineDev) [inline source/dlangbot/github.d]: I think `"dlang/undeaD", "dlang/dconf.org"` are also good candidates for the list.

File: source/dlangbot/github.d (hunk 1 above)
Comment (ZombineDev) [inline source/dlangbot/github.d]: Ugh, so plain newline doesn't work?

File: source/dlangbot/github.d (hunk 3 above)
Comment (wilzbach) [inline source/dlangbot/github.d]: The Bugzilla integration isn't set up for these repos and they aren't looked at by the changelog generator.

File: source/dlangbot/github.d (hunk 1 above)
Comment (wilzbach) [inline source/dlangbot/github.d]: Yep :/

File: source/dlangbot/github.d (hunk 2 above)
Comment (wilzbach) [inline source/dlangbot/github.d]: Nope. Once you are in HTML, there is no way back..

File: source/dlangbot/github.d (hunk 3 above)
Comment (ZombineDev) [inline source/dlangbot/github.d]: Ah, ok.

**Pull request merged**